pydss/flapper.py

```diff
@@ -33,16 +40,9 @@ def move(self, adr,amt):
 */
 """
 class Flapper:
-    wards = dict()
-    #messageclass
-    message = 0;
-
-    def rely(self, usr,send):
-        if self.wards[send] != 1:
-            self.wards[usr] = 1
-    def deny(self, usr,send):
-        if self.wards[send] != 1:
-            self.wards[usr] = 0
+
+
+
```

Comment:
could you add documentation for the instance variables in the convention we've set up? here's an example from `Flipper`:

```
class Flipper:
    """
    ADDRESS = str
    beg = Wad
    bids = dict[int: Bid]
    cat = Cat
    ilk_id = str
    kicks = int
    ttl = int
    tau = int
    vat = Vat
    """
```